.travis.yml

```diff
@@ -1,58 +1,51 @@
 language: python
-python: 2.7
 sudo: false
+
 cache:
   pip: true
   directories:
     - $HOME/buildout-cache
+
 before_install:
-  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
-  - mkdir $HOME/.buildout
-  - echo "[buildout]" > $HOME/.buildout/default.cfg
-  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
-  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
-  - pip install --upgrade pip setuptools zc.buildout
-  - if [[ `npm -v` != 3.10.7 ]]; then npm i -g npm@3.10.7; fi
-  - npm install -g grunt-cli
-
-env:
-  matrix:
```

Comment:
Sorry @tisto but this way of running tests is extrem hard to understand on hard to extend. I have replaced that with a way that could solve that much more straight forward and still be readable.

To be in detail:
We should distinguish between this package itself (bobtemplates.plone) which should be tested, and all of its generated  packages. 
This package itself did not have any direct add-on relation to Plone, so testing against different Plone versions for this package makes no sense.
The generated packages, should be compatible with multiple Plone versions, so they need a useful test matrix to test against. Please look at https://github.com/plone/bobtemplates.plone/blob/tox/tests/test_skeleton_generation.py#L9-L83 for that.

My second point is, that tests should be runnable local and on CI without any difference, so I did move all the parts that are test related into the test-runner / test invocation part.

If all generated products would have have a tox.ini it would be much easier to test and maintain.

Any additional Feedback is welcome.